README.md

```diff
@@ -281,8 +281,7 @@ If you get this error:
 
 ### 2 Factor Authentication / App Passwords
 
-The Error: "The setting you are looking for is not available for your account."
-will occur if you do not have 2 Factor Authentication (2FA) enabled on your Google account.
+The Error: "The setting you are looking for is not available for your account." will occur if you have 2 Factor Authentication (2FA) enabled on your Google account.
```

Comment:
This is not an error. This means that you can't create app password if you don't have 2FA enabled. Style can be slightly improved though.
```suggestion
The error "The setting you are looking for is not available for your account" will occur if you do not have 2 Factor Authentication (2FA) enabled on your Google account.
```